src/docs/010_manual/20_install.adoc

```diff
@@ -184,9 +197,9 @@ build
         `-- arc42-logo.png
 ----
 
-*Congratulations!* If you see the same folder structure, you just managed to render the standard arc42 template as html and pdf!
+*Congratulations!* If you see the same folder structure, you've just render the standard arc42 template as html and pdf!
```

Comment:
Congratulations! If you see the same folder structure, you've just rendered the standard arc42 template as HTML and PDF.